odoo/addons/base/migrations/10.0.1.3/pre-migration.py

```diff
@@ -57,5 +57,7 @@ def migrate(cr, version):
     cr.execute(
         '''update ir_module_module
         set state = 'to remove'
-        where name in ('web_tip', 'web_view_editor')
+        where name in (
```

Comment:
I'm seeing that a better technique is to run `update_module_names` with `merge_modules=True` option. I'm trying to get a green branch on https://github.com/OCA/OpenUpgrade/pull/837